src/types/modifications/Adjust.jl

```diff
@@ -299,12 +311,13 @@ function adjust_by_age!(config, data, row)
     hasproperty(table, variable_name) || error("Table $table_name has no column $variable_name to adjust in `adjust_hourly!`")
 
     # Perform the adjustment on each row of the table
+    row.num_adjusted = nrow(table)
```

Comment:
how do we use num_adjusted? Is it just something we can check later in the data?